FirebaseAppDistribution/Sources/FIRAppDistributionUIService.m

```diff
@@ -168,15 +168,37 @@ - (void)initializeUIState {
   if (self.window) {
     return;
   }
-  // Create an empty window + viewController to host the Safari UI.
+
+#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
+  if (@available(iOS 13.0, *)) {
+    UIWindowScene *foregroundedScene = nil;
+    for (UIWindowScene *connectedScene in [UIApplication sharedApplication].connectedScenes) {
+      if (connectedScene.activationState == UISceneActivationStateForegroundActive) {
+        foregroundedScene = connectedScene;
+        break;
+      }
+    }
+
+    if (foregroundedScene) {
+      self.window = [[UIWindow alloc] initWithWindowScene:foregroundedScene];
+    } else {
+      // TODO: figure out if foregroundScene can be nil. If yes, should we return error?
+      return;
+    }
+  }
+#else  // defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
   self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+#endif
   self.window.rootViewController = self.safariHostingViewController;
 
   // Place it at the highest level within the stack.
   self.window.windowLevel = +CGFLOAT_MAX;
 
   // Run it.
   [self.window makeKeyAndVisible];
+
+  if (self.window) {
```

Comment:
Do we need this?

Reply:
Nope. sorry was trying something else and didnt delete this.

Reply:
Removed.